assets/js/app/Instance.js

```diff
@@ -18,46 +18,57 @@ LocusZoom.Instance = function(id, datasource, layout, state) {
     this.initialized = false;
 
     this.id = id;
-    this.parent = LocusZoom;
     
     this.svg = null;
 
-    // The _panels property stores child panel instances
-    this._panels = {};
+    // The panels property stores child panel instances
+    this.panels = {};
     this.remap_promises = [];
+
+    // The layout is a serializable object used to describe the composition of the instance
+    this.layout = layout || JSON.parse(JSON.stringify(LocusZoom.DefaultLayout));
```

Comment:
I would actually lean the other way in that layout would only be used to "inflate" the object. Other important state would be maintained in classes/variables that can be serialized on demand. The problem with a live layout is that you have to constantly watch for any property changes. And if you create objects based on values in the layout, you'd have to potentially rebuild them everytime. You've been working with the layout far more actively so if you think the living layout would be better long term, i'll trust your judgement.